cohort-engine/src/main/java/com/ibm/cohort/engine/measure/seed/MeasureEvaluationSeeder.java

```diff
@@ -45,6 +44,8 @@
 
 	private boolean enableExpressionCaching;
 	private boolean debugMode = true;
+	
+	private static final String MEASUREMENT_PERIOD = "Measurement Period";
```

Comment:
The CMS and HEDIS measures that I've looked at use this name for the measure period parameter, but it isn't defined anywhere in the specifications that I've seen. It makes me a bit nervous hard-coding it and then just expecting that the CQL author's will use it with that name. At one point I thought that Dan had made it configurable, but maybe that never happened. If we are forcing this specific name for the Measurement Period, I would like to see it at least documented in our user guide.